ui/apps/platform/src/Containers/Policies/Table/PoliciesTable.tsx

```diff
@@ -453,14 +453,7 @@ function PoliciesTable({
                                         />
                                     </Td>
                                     <Td dataLabel="Status">
-                                        {disabled ? (
-                                            'Disabled'
-                                        ) : (
-                                            <Flex className="pf-u-info-color-200">
-                                                <CheckCircleIcon className="pf-u-mr-sm pf-m-align-self-center" />
-                                                Enabled
```

Comment:
Did you mean to remove the check-circle, too?

Reply:
Good eyes. Yes, `PolicyDisabledIconText` renders an distinct icon for both Disabled and Enabled.